build/org.eclipse.pde.build.tests/pom.xml

```
@@ -0,0 +1,89 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project
+	xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 https://maven.apache.org/xsd/maven-4.0.0.xsd"
+	xmlns="http://maven.apache.org/POM/4.0.0"
+	xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance">
+	<modelVersion>4.0.0</modelVersion>
+	<parent>
+		<groupId>org.eclipse.pde</groupId>
+		<artifactId>eclipse.pde.build</artifactId>
+		<version>4.31.0-SNAPSHOT</version>
+	</parent>
+	<artifactId>org.eclipse.pde.build.tests</artifactId>
+	<version>1.4.300-SNAPSHOT</version>
+	<packaging>eclipse-test-plugin</packaging>
+
+	<properties>
+		<!-- This is sued in the ibuilds where the tycho test execution is
+		DISABLED and instead we rely on the test.xml! -->
+		<skipTests>true</skipTests>
+		<ignoredWarnings>${tests.ignoredWarnings}</ignoredWarnings>
+	</properties>
+
+	<profiles>
+		<profile>
+			<id>local-pde-tests</id>
```

Can't we just re-use `build-individual-bundles` here?
```suggestion
			<id>build-individual-bundles</id>
```

---

Yes we can!

---

done